sonar-docs/analysis/scan/sonarscanner-for-azure-devops.md

```diff
@@ -148,17 +148,30 @@ When a build is run on a branch of your project, the extension automatically con
 If you are working with branches on TFVC projects, you still need to manually specify the branch to be used on {instance}: in the **Prepare Analysis Configuration** task, in the **Additional Properties**, you need to set `sonar.branch.name`.
 
 ### PRs
-{instance} can analyze the code of the new features and annotate your pull requests in TFS with comments to highlight issues that were found.
+{instance} can analyze the code of the new features and annotate your pull requests in Azure DevOps with comments to highlight issues that were found.
 
 Pull request analysis is supported for any type of Git repositories. To activate it:
 
 1. In the **Branch policies** page of your main development branches (e.g. "master"), add a build policy that runs your build definition
 1. Create an Azure DevOps token with "Code (read and write)" scope
-1. <!-- sonarqube -->In SonarQube, in the **[Administration > General Settings > Pull Requests](/#sonarqube-admin#/admin/settings?category=pull_request)** page,<!-- /sonarqube --><!-- sonarcloud -->In SonarCloud,<!-- /sonarcloud --> set this token in the **VSTS/TFS** section
+1. <!-- sonarqube -->In SonarQube, in the **[Administration > General Settings > Pull Requests](/#sonarqube-admin#/admin/settings?category=pull_request)** page,<!-- /sonarqube --><!-- sonarcloud -->In SonarCloud,<!-- /sonarcloud --> set this token in the **Azure DevOps** section
 
 Next time some code is pushed in the branch of a pull request, the build definition will execute a scan on the code and publish the results in {instance} which will decorate the pull request in TFS.
 
-
+<!-- sonarcloud -->
+## Using Release Pipelines
+You have the possibility to check the SonarCloud quality gate status in your release pipeline. It takes place as a [pre-deployment gate](https://docs.microsoft.com/en-us/azure/devops/pipelines/release/approvals/gates?view=azure-devops).
+
+1. In the **release pipeline**, add a stage, then click on **pre-deployment conditions**.
+2. Enable the **gates**, then click on add. Choose **SonarCloud QualityGate status check**
+3. Save your pipeline.
+
+**This feature is currently in preview, and the following notes are important** :
+* The **publish quality gate result** task has to be enabled in order to get this gate working.
```

Comment:
Should probably say that the `Publish Quality Gate Result` is in the build pipeline used to create the artefacts, just to avoid any misunderstanding.

Also, should probably use the correct capitialisation for the task name rather than being all lower-case.

Reply:
Yes of course, this has been modified.